docs/devices/9290012573A.md

```diff
@@ -48,10 +48,9 @@ With [one](./324131092621.md) of the [two](./929002398602.md) Hue Dimmer switche
 
 See also the [VIDEO: Factory reset a Hue bulb with Hue dimmer switch](https://www.youtube.com/watch?v=qvlEAELiJKs).
 
-#### Bluetooth (if supported by device)
-Install the Philips Hue Bluetooth app for [Android](https://play.google.com/store/apps/details?id=com.signify.hue.blue)
-or [iOS](https://apps.apple.com/us/app/philips-hue-bluetooth/id1456604186). You can use the app to trigger a factory reset on a paired light. (Note: The light will only be in bluetooth pairing
-mode for a couple of minutes after power-on)
+#### Bluetooth (unsupported here)
```

Comment:
I think it would be better to remove the bluetooth section entirely.

Reply:
Maybe put it in a different place but the hint that BT is definitively not available could be handy so that people don't keep trying

Reply:
Then I suggest to remove it and put something like: "This device does not support Bluetooth right under" "## Notes"

Reply:
✓